### PR TITLE
DROTH-338: include TR equipment data in Integration API

### PR DESCRIPTION
**Pull request by tvenhola**
Include TR equipment on integration api

### CODE REVIEW

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/MassTransitStopService.scala [hunk 1]
```diff
@@ -86,24 +86,33 @@ trait MassTransitStopService extends PointAssetOperations {
     }
   }
 
+  /**
+    * Run enriching if the stop is found in Tierekisteri. Return enriched stop with a boolean flag for errors (not found in TR)
+    * @param persistedStop Optional mass transit stop
+    * @return Enriched stop with a boolean flag for TR operation errors
+    */
+  private def enrichStopIfInTierekisteri(persistedStop: Option[PersistedMassTransitStop]) = {
```
Comment (marquesfm) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/MassTransitStopService.scala]: shouldnt we specify the returning type?

Comment (tvenhola) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/MassTransitStopService.scala]: It's defined by what it's returning and type resolution works then automatically.

**Comment by marquesfm** [comment]
Ok for me.

**Comment by leirasf** [comment]
It's ok to me

**Pull request merged**